Tests/instance_notifier.py

```diff
@@ -24,6 +24,11 @@ def options_handler():
     return options
 
 
+def install_new_content(client, server):
+    prints_manager = ParallelPrintsManager(0)
+    update_content_on_demisto_instance(client, server, prints_manager, 0)
```

Comment:
print the collected messages after calling the function:
`prints_manager.execute_thread_prints(0)`

Reply:
True